src/app/features/home/post-capture-tab/post-capture-tab.component.spec.ts

```diff
@@ -9,15 +10,16 @@ describe('PostCaptureTabComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       declarations: [PostCaptureTabComponent],
-      imports: [SharedTestingModule],
+      imports: [SharedTestingModule, VirtualScrollerModule],
     }).compileComponents();
 
     fixture = TestBed.createComponent(PostCaptureTabComponent);
     component = fixture.componentInstance;
     fixture.detectChanges();
   }));
 
-  it('should create', () => {
+  // FIXME: May be related to https://github.com/numbersprotocol/capture-lite/issues/277
```

Comment:
I don't think it is related to the issue you mentioned as the issue has been fixed by removing the `ngStyle` introduced by DaYuan. We can still exclude this test for now. However, this comment should also be removed to avoid possible misunderstanding.